kapitan/inputs/binary.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 The Kapitan Authors
+# SPDX-FileCopyrightText: 2020 The Kapitan Authors <kapitan-admins@googlegroups.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import re
+import logging
+import os
+import shutil
+from distutils.dir_util import copy_tree
+
+from kapitan.inputs.base import InputType
+
+logger = logging.getLogger(__name__)
+
+
+class Binary(InputType):
+    def __init__(self, compile_path, search_paths, ref_controller):
+        super().__init__("binary", compile_path, search_paths, ref_controller)
+        self.binary_env_vars = {}
+        self.binary_args = []
+        self.target_name = None
+
+    def set_binary_env_vars(self, binary_env_vars):
```

Comment:
`set_env_vars()` seems clearer to me. Same for `binary_env_vars`, `env_vars` is cleaner?